flowzone.yml

```
@@ -1617,12 +1625,20 @@ jobs:
           node-version: 18
 
       - name: Docusaurus Builder
-        uses: product-os/docusaurus-builder@master
+        if: |
+          needs.project_types.outputs.has_readme == 'true' &&
+          inputs.docusaurus_website != 'true'
+        uses: product-os/docusaurus-theme@9cb615b78bcf2c2f9d8fa683a296a52d7beefae1 # 1.3.0
         with:
           repo: ${{ github.event.repository.name }}
           org: ${{ github.repository_owner }}
           default_branch: ${{ github.event.repository.default_branch }}
-          url: https://${{ inputs.cloudflare_website }}.pages.dev
+          url: https://${{ inputs.cloudflare_website }}.pages.dev/
+
+      - name: Custom Website Builder
+        if: |
+          inputs.docusaurus_website == 'true'
```

```suggestion
          inputs.docusaurus_website != true
```
I think inputs are actual booleans, and not strings like outputs.

---

Really, I see strings around all booleans everywhere for some reason.